src/i18n/locale/en.json

```diff
@@ -14,6 +14,297 @@
     "DownloadQueue": {
       "no-downloads": "No downloads",
       "download-queue": "Download Queue"
+    },
+    "SourceMangas": {
+      "no_mangas_found": "No manga was found!",
+      "local_source": {
+        "label": {
+          "checkout": "Check out",
+          "guide": "Local source guide"
+        }
+      }
+    },
+    "settings": {
```

Comment:
be consistent in capitalization

Reply:
thats the sub folder inside "screen"
do you think it would be better to omit it and just have the screens inside it be placed directly in "screens"?

Reply:
how about `screens.settings.`

Reply:
thats the current key